packages/core/src/components/html-table/_overrides.scss

```diff
@@ -0,0 +1,84 @@
+// Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+
+@import "../../common/variables";
+@import "../../common/_customs";
+
+
+
+table.#{$ns}-html-table {
+
+  background-color: $g-light-base3;
+
+  thead tr:last-child,
+  tbody tr:last-child,
```

Comment:
Instead of adding a bottom border to tbody & tfooter, we can add only a top border to tfooter. This way, we avoid a border at the end of the table in tables without footer.

Reply:
Changed!